quest/regression/quest_intersection_shaper/case1_012.yaml

```diff
@@ -0,0 +1,7 @@
0, 0.999999999999999, 0.999999999999999, 1.0, 1.0, 1.0, 1.0, 1.0, 0.999999999999999, 0.999999999999998, 1.0, 1.0, 1.0, 0.999999999999999, 0.999999999999998, 1.0, 0.999999999999998, 1.0, 0.999999999999998, 1.0, 1.0, 0.999999999999998, 1.0, 0.999999999999998, 0.999999999999998, 0.999999999999996, 0.999999999999999, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.999999999999998, 1.0, 1.0, 1.0, 0.999999999999999, 1.0, 1.0, 0.999999999999999, 1.0, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999999, 0.999999999999999, 0.999999999999999, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999998, 1.0, 0.999999999999997, 1.0, 0.999999999999999, 0.999999999999998, 0.999999999999997, 0.999999999999998, 1.0, 1.0, 1.0, 1.0, 0.999999999999999, 1.0, 1.0, 0.999999999999997, 1.0, 0.999999999999999, 0.999999999999999, 1.0, 0.999999999999998, 1.0, 1.0, 0.999999999999998, 1.0, 0.999999999999999, 1.0, 0.999999999999999, 0.999999999999999, 1.0, 0.999999999999999, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.999999999999999, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.999999999999999, 0.999999999999999, 1.0, 0.999999999999999, 0.999999999999999, 1.0, 1.0, 0.999999999999998, 1.0, 0.999999999999999, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.999999999999999, 0.999999999999998, 1.0, 1.0, 0.999999999999998, 0.999999999999999, 1.0, 0.999999999999998, 0.999999999999998, 0.999999999999998, 0.999999999999999, 0.999999999999998, 1.0, 0.999999999999998, 1.0, 0.999999999999999, 0.999999999999998, 1.0, 0.999999999999996, 0.999999999999999, 0.999999999999995, 1.0, 1.0, 0.999999999999998, 0.999999999999998, 1.0, 0.999999999999999, 0.999999999999998, 0.999999999999999, 1.0, 0.999999999999997, 0.999999999999997, 1.0, 0.999999999999999, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999998, 1.0, 1.0, 0.999999999999997, 0.999999999999999, 1.0, 0.999999999999998, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999999, 0.999999999999999, 0.999999999999998, 0.999999999999999, 0.999999999999999, 1.0, 0.999999999999997, 0.999999999999998, 0.999999999999999, 0.999999999999998, 1.0, 1.0, 1.0, 1.0, 1.0, 0.735266532531401, 0.735266532531404, 0.735266532531404, 0.735266532531404, 0.735266532531404, 0.735266532531405, 0.735266532531401, 0.735266532531403, 0.735266532531404, 0.735266532531404, 0.735266532531404, 0.735266532531405, 0.735266532531404, 0.735266532531405, 0.735266532531403, 0.735266532531405, 0.735266532531402, 0.735266532531404, 0.735266532531404, 0.735266532531402, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
```

Comment:
Thanks for adding this regression data!

Would it make sense for these to be in a binary format (e.g. `hdf5`) rather than ascii?

Reply:
I initially was using the MFEMSidreDataCollection to write the data but it did not like to read it back in for some reason. It insisted the volume fraction grid functions were invalid even though it wrote them. None of the *DataCollection options really were behaving 100% for me so I opted to write directly via Conduit and then YAML so because I wanted to be able to eyeball the values.